package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A server side implementation to serve a First Discovery Editor instance and provide a means for storing preferences to the GPII Preferences server.",
   "main": "index.js",
   "engines": {
-    "node": ">=0.10.0"
+    "node": "0.10.36",
```

Comment:
I think being forgiving on versions of node and npm (at least to the extent of 0.10.x and 2.x.x) is reasonable - even if we are deciding to be less forgiving about code dependencies